skimage/data/_registry.py

```diff
@@ -25,6 +25,8 @@
     'logo.png',
     'microaneurysms.png',
     'moon.png',
+    'multipage_rgb.tif',
```

Comment:
```suggestion
    'multipage.tif',
    'multipage_rgb.tif',
```
?

Also, the list looks like it's sorted alphabetically but it's not (e.g., we have `'motorcycle_*.png'` at the very end).